lib/mongodb.js

```diff
@@ -180,6 +188,21 @@ MongoDB.prototype.fromDatabase = function (model, data) {
         // Convert the Binary into String
         data[p] = data[p].toString();
       }
+    }  // check if prop contains EmbedsMany
+    else if(prop && prop.type.constructor === Array &&
```

Comment:
There is a better way to check relations. You can access modelClass.definition.relations.